xbmc/video/VideoDatabase.cpp

```diff
@@ -6451,6 +6451,7 @@ bool CVideoDatabase::GetSetsByWhere(const std::string& strBaseDir, const Filter
 
     CFileItemList sets;
     GroupAttribute groupingAttributes;
+    int groupRating = CServiceBroker::GetSettingsComponent()->GetSettings()->GetInt(CSettings::SETTING_VIDEOLIBRARY_MOVIESETRATING);
```

Comment:
It seems the value of `groupRating`is not used in this member function. This line should be removed. 

Or did you mean to add the value to the call to `GroupUtils::Group` in line 6469 and forgot to add the new parameter?